src/emulator/modules/SceAppUtil/SceAppUtil.cpp

```diff
@@ -273,16 +273,15 @@ EXPORT(int, sceAppUtilStoreBrowse) {
 }
 
 EXPORT(int, sceAppUtilSystemParamGetInt, unsigned int paramId, int *value) {
-    auto sys_lang = static_cast<SceSystemParamLang>(host.cfg.sys_lang);
-    if (sys_lang > SCE_SYSTEM_PARAM_LANG_TURKISH)
```

Comment:
I think this check should still be present for safety. User may edit the value directly from config file, and it may be invalid. Can put it somewhere if you like, but here is good.

Reply:
yesd, i have set it before because user need set manually value, but now is not needed ^^